fdbclient/admin_client.go

```diff
@@ -201,6 +202,51 @@ func (client *cliAdminClient) runCommand(command cliCommand) (string, error) {
 	return outputString, nil
 }
 
+// CleanupOldLogs removes old fdbcli log files.
+func (client *cliAdminClient) CleanupOldLogs() error {
+	logDir := os.Getenv("FDB_NETWORK_OPTION_TRACE_ENABLE")
+	if logDir == "" {
+		return nil
+	}
+
+	err := filepath.Walk(logDir, func(path string, info os.FileInfo, err error) error {
+		if err != nil {
+			return err
+		}
+
+		if info.IsDir() {
+			return nil
+		}
+
+		if !strings.HasPrefix(info.Name(), "trace") {
+			return nil
+		}
+
+		// If the file is newer than 1 hour skip it
+		if info.ModTime().Add(1 * time.Hour).Before(time.Now()) {
```

Comment:
Thinking more about this we could also add a flag for this to make it configurable.

Reply:
I agree this should be configurable.